app/src/main/java/com/example/workoutapp/dao/IWorkoutRecDAO.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.workoutapp.dao
+
+import com.example.workoutapp.dto.WorkoutRec
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface IWorkoutRecDAO {
+    @GET("")
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere? 

Recommendation: if not used, remove it, or keep it in a separate development branch until it is complete.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.